.github/workflows/integration.yaml

```diff
@@ -291,34 +289,6 @@ jobs:
             -f ../.github/wasi_testsuite_skip.json \
             -r ../.github/wasi_testsuite_adapter.py
 
-  build_gojs_test_binaries:
```

Comment:
I stopped this because it only takes 2s to build the test binary, and the cache setup was distracting.